<ui version="4.0" >
 <class>FileSelector</class>
 <widget class="QDialog" name="FileSelector" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>486</height>
   </rect>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBoxFiles" >
      <property name="title" >
       <string>Files</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QCheckBox" name="checkSelectAll" >
         <property name="text" >
          <string>Select / deselect &amp;all</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QTreeView" name="treeViewFiles" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights" >
          <bool>true</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBoxLogMessage" >
      <property name="title" >
       <string>Log Message</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QTextEdit" name="editLogMessage" />
       </item>
       <item row="0" column="0" >
        <widget class="QComboBox" name="comboLogHistory" >
         <property name="maxCount" >
          <number>15</number>
         </property>
         <property name="sizeAdjustPolicy" >
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonHelp" >
       <property name="text" >
        <string>&amp;Help</string>
       </property>
       <property name="shortcut" >
        <string>F1</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>41</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QSplitter" name="splitter_3" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>444</width>
     <height>352</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <action name="actionRevert" >
   <property name="icon" >
    <iconset resource="../qsvn.qrc" >:/images/actionrevert.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Revert</string>
   </property>
  </action>
  <action name="actionDiff" >
   <property name="text" >
    <string>Diff</string>
   </property>
  </action>
  <action name="actionResolved" >
   <property name="text" >
    <string>Resolved</string>
   </property>
   <property name="iconText" >
    <string>Resolved</string>
   </property>
   <property name="toolTip" >
    <string>Resolved</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qsvn.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FileSelector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>446</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
